### PR TITLE
Update to version 1.9.2.0 of the geometry dictionary

### PR DESCRIPTION
**Pull request by mit3ch**
## Summary

- Added enumerated values to coordinate_space_frame_type to support Mars2020:
  - CINT_FRAME
  - DRILL_BIT_TIP
  - MEDA_RDS
  - SUN
  - ROVER_FRAME
  - TOOL_FRAME
  - Tool_Frame
- Corrected a typo in an enumerated value for coordinate_space_frame_type:
  from RSM_Head_FramE to RSM_Head_Frame.

resolves https://github.com/pds-data-dictionaries/PDS4-LDD-Issue-Repo/issues/151
resolves https://github.com/pds-data-dictionaries/PDS4-LDD-Issue-Repo/issues/146
resolves https://github.com/pds-data-dictionaries/PDS4-LDD-Issue-Repo/issues/145

### CODE REVIEW

**Review by jordanpadams — APPROVED** [review]

**Comment by jordanpadams** [comment]
@mit3ch I approved this change and updated the branch to master. it looks like you were using an old `dev-mkg` branch. i would highly recommend start with a fresh, new branch off of master whenever you start making a change. we can merge these changes and add @edguinness' changes to another branch? Or do we want to just wait for those changes as well?

**Comment by mit3ch** [comment]
@jordan I charged ahead before reading your comment. Sorry. I will start a fresh branch next time. I was still working in dev-mkg and added  @edguinness' changes to that and did a pull request (I think). It didn't behave quite like I was expecting. It looks like the new commit was slurped into the pull request you approved earlier today. So I edited the Comment to include the three additional enumerated values (ROVER_FRAME, TOOL_FRAME, and Tool_Frame). If that really is what happened, this can be merged with Master. When the dictionary run for the new build happens, we need three variations of this version (1920) of the dictionary. One each for IM 1.4.0.0, 1.5.0.0, and 1.6.0.0. The file pds4_versions.txt for this dictionary already lists the first two. Do I need to do something to keep them from being overwritten by 1.6.0.0?

File: src/PDS4_GEOM_IngestLDD.xml [hunk 1]
```diff
@@ -2716,6 +2731,11 @@
              <value_meaning>Frame describing the movable sensor head of the PIXL instrument.</value_meaning>
             </DD_Permissible_Value>
             
+            <DD_Permissible_Value>
+             <value>ROVER_FRAME</value>
+             <value_meaning>TBD</value_meaning>
```
Comment (jordanpadams) [inline src/PDS4_GEOM_IngestLDD.xml:2736]: @mit3ch @edguinness did we want to put a definition here before we merge?

File: src/PDS4_GEOM_IngestLDD.xml [hunk 2]
```diff
@@ -2737,11 +2757,22 @@
...
+            
+            <DD_Permissible_Value>
+                <value>SUN</value>
+                <value_meaning>Frame centered on the sun. Generally used for pointing instruments at the sun,
+                    rather than 3D position.</value_meaning>
+            </DD_Permissible_Value>
+            
+            <DD_Permissible_Value>
+             <value>TOOL_FRAME</value>
+             <value_meaning>TBD</value_meaning>
```
Comment (jordanpadams) [inline src/PDS4_GEOM_IngestLDD.xml:2772]: @mit3ch @edguinness same question here per a definition

File: src/PDS4_GEOM_IngestLDD.xml [hunk 3]
```diff
@@ -2914,15 +2945,22 @@
...
             </DD_Permissible_Value>
             
             <DD_Permissible_Value>
              <value>Site_Frame</value>
              <value_meaning>Instance of LOCAL_LEVEL_FRAME that is fixed to the ground. Used for local operations to reduce error propagation due to position uncertainty.</value_meaning>
             </DD_Permissible_Value>
+
+            <DD_Permissible_Value>
+             <value>Tool_Frame</value>
+             <value_meaning>TBD</value_meaning>
```
Comment (jordanpadams) [inline src/PDS4_GEOM_IngestLDD.xml:2959]: @mit3ch @edguinness same question here per a definition

**Comment by jordanpadams** [comment]
@mit3ch per your question, 1.16.0.0 will be added to all of the repos when we tag a release. I am just waiting for the last minute SCR to get approved and implemented before building anything.

**Comment by edguinness** [comment]
I asked IDS for definitions, but they haven't provided ones for the new values.

File: src/PDS4_GEOM_IngestLDD.xml (hunk 1 above)
Comment (jordanpadams) [reply src/PDS4_GEOM_IngestLDD.xml:2736]: @rgdeen ☝️

**Comment by jordanpadams** [comment]
@mit3ch just as a heads up, one nice thing about Github is you can [link issues to pull requests](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), which not only helps trace issues to implementation, but it also closes tickets out for you whenever the pull request is merged. i've updated this pull request description to include this.

**Comment by jordanpadams** [comment]
@mit3ch @edguinness I pinged @rgdeen offline and got the updated the definitions

**Pull request merged**